ulauncher/ui/windows/preferences/utils/ext_handlers.py

```diff
@@ -99,7 +99,7 @@ def install_async() -> None:
             try:
                 ext = asyncio.run(ExtensionController.install(url))
```

Comment:
unrelated to this PR, but would be good to drop asyncio completely to simplify the code.

Reply:
Yeah. We talked about this. I am not at all convinced that it will simplify the code. I think it will simplify some places and do the opposite in a bunch of other places. It might still be an overall improvement though.